spec/kitchen/driver/vagrant_spec.rb

```diff
@@ -77,6 +87,66 @@
       Kitchen::Driver::VAGRANT_VERSION)
   end
 
+  describe "#run_command" do
+    before(:each) do
+      allow(driver_object).to receive(:path_elem_separator).and_return(":")
+    end
+
+    context "when invoked from a clean environment" do
+
+      it "passes through environment variables" do
+        options = driver.send(
```

Comment:
Why `driver.send` instead of just `driver.run_command`?  It didn't look like it was private...